inst/qml/raincloudPlots.qml

```diff
@@ -0,0 +1,900 @@
+				label:      		""
+				enabled: 			covariate.count === 1
+				indexDefaultValue:  3
+				infoLabel:			qsTr("Covariate palette")
+				info:  				qsTr("How to color code the covariate. 'Viridis' works good for both discrete and continuous covariates.")
+			}
+		}
+
+		CheckBox
+		{
+			name: 					"colorAnyway"
+			label:					qsTr("Apply color palette to primary factor")
+			id: 					colorAnyway
+			enabled: 				secondaryFactor.count === 0
+			checked:				secondaryFactor.count === 0
+			Layout.columnSpan: 		2
+			info:					qsTr(
+										"Applies the color palette to the levels of the primary factor. " +
+										"Otherwise, the plot stays black and white.<br>" +
+										"This option is superseeded by a secondary factor; then color coding is according to that."
+									)
+		}
+
+		// End top 2 rows
+		
+		CheckBox
+		{
+			name: 				"horizontal"
+			label: 				qsTr("Horizontal plot")
+			Layout.columnSpan:  3
+			info:				qsTr(
+									"Plots the dependent variable axis horizontally, at the bottom of the plot.<br>" +
+									"This is where the raincloud plot gets its name from: It will look like the points are raining from the violin and box (cloud)."
+								)
+		}
+
+		CheckBox
+		{
+			name:   			"table"
+			label:  			qsTr("Table with statistics")
+			info:				qsTr(
+									"Shows a table under the plot with statistics per cloud, like number of observations or median.<br>" +
+									"If you select 'Mean' or 'Interval around mean' (see Advanced Section), they will also be shown here."
+								)
+			CheckBox
+			{
+				name:  			"tableBoxStatistics"
+				label: 			qsTr("Box Statistics")
+				checked: 		true
+				info:			qsTr(
+									"Shows the box statistics in the table: lower whisker, 25th percentile, median, 75th percentile, upper whisker.<br>" +
+									"It can be helpful to un-check this if you are also working with 'Interval around mean' (see Advanced Section)."
+								)
+			}
+		}
+
+	}  // End section General
+
+
+
+	Section
+	{
+		title:   qsTr("Cloud Elements")
+		columns: 3
+
+		CheckBox
+		{
+			name: 		"showVio"
+			id: 		showVio
+			text: 		qsTr("Show violin")
+			checked: 	true
+			info: 		qsTr(
+							"Whether or not the violin should be shown. If un-checked, opacity is set to 0 and outline to 'none'."
+						)
+		}
+		CheckBox
+		{
+			name: 		"showBox"
+			id:   		showBox
+			text: 		qsTr("Show box")
+			checked: 	if ( meanInterval.checked || meanIntervalCustom.checked )
+						{
+							false
+						} else {
+							true
+						}
+			info: 		qsTr(
+							"Whether or not the box should be shown. If un-checked, opacity is set to 0 and outline to 'none'."
+						)
+		}
+		CheckBox
+		{
+			name: 		"showPoint"
+			id: 		showPoint
+			text: 		qsTr("Show point")
+			checked: 	true
+			info: 		qsTr(
+							"Whether or not the points of a cloud should be shown. If un-checked, opacity is set to 0.<br>" +
+							"If you have many, many points, it can be helpful to hide them." +
```

Comment:
@JorisGoosen I don't think this is easier tot translate, no?

Reply:
I do not fully understand this comment and will wait for Joris´ response.

Reply:
Would it be okay to leave it as is and to integrate the rainclouds into one of the upcoming nightlies?